### ISSUE TITLE
diameter size in xenium tutorial

### ISSUE THREAD

**Issue by revolvefire** [BiomedicalMachineLearning/stLearn#250]
Thanks for the wonderful tool!

I've been working on the xenium data
and I wasn't 100% sure what exactly "spot_diameter_fullres" means in the xenium tutorial

Does the number for this parameter means
the diameter of the spot(circle) from which all the transcripts are taken into account as those for a cell?

So from an x,y coordinate that xenium provides for each cell (that it thinks is a cell, based on the DAPI image)
stlearn will take into account all the transcript within the circle with a diameter of 15um (perhaps the center would be the exact x,y coordinate)?
am I getting at it correctly?

Thanks!

**Comment by duypham2108** [comment]
It does not mean that actually. The Xenium (computational) protocol did the transcript count already. The "spot_diameter_fullres" parameter serve for other automatic calculation like neighborhood finding in CCI analysis. You don't need to worry about it

**Comment by SGIlabes** [comment]
Thank you for creating such a wonderful package. I am finding it very useful. 
And I have a similar question. The term 'spot_diameter_fullres' is described as 'The number of pixels that span the diameter of a  spot in the original, full-resolution image.' (in 10X visium). So I'm wondering, is the reason for setting 'spot_diameter_fullres' to 15 for the convenience of computer calculations? Theoretically, should the 'spot_diameter_fullres' be 1 instead?

**Comment by duypham2108** [comment]
Yes, we set 'spot_diameter_fullres' to 15 just for convenience, but it should not be used in Xenium data. The term 'spot_diameter_fullres' is created for spot-based data only. It could be useful only if you bin the Xenium data and the value should be your binning size

**Comment by SGIlabes** [comment]
Thanks!

> Yes, we set 'spot_diameter_fullres' to 15 just for convenience, but it should not be used in Xenium data. The term 'spot_diameter_fullres' is created for spot-based data only. It could be useful only if you bin the Xenium data and the value should be your binning size
